dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05360AddDBSchedulerTablesTest.java

```diff
@@ -0,0 +1,45 @@
+package com.dotmarketing.startup.runonce;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+
+public class Task05360AddDBSchedulerTablesTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+    
+    @Test
+    public void test() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)